### ISSUE TITLE
ACE3 Advanced Medical System

### ISSUE THREAD

**Issue by freghar** [CntoDev/mods#3]
_From @ghost on September 27, 2015 17:38_

Following a test of the ace3mod's AMS, I have uploaded a video of a small post-mission consensus from all the members who attended.

[![Click to View](http://img.youtube.com/vi/qWKVWUnnnKg/0.jpg)](http://www.youtube.com/watch?v=qWKVWUnnnKg)

_Copied from original issue: CntoDev/cnto-mmt-repo#2_

**Comment by freghar** [comment]
_From @ghost on September 27, 2015 17:40_

Next step would be to play another mini-op where this system was implemented into a mission following our official template. I will be proposing this to @JamesTheClarke at the MM Meeting today.

**Comment by freghar** [comment]
_From @ghost on October 3, 2015 16:34_

Scheduled next test for the 15th, will be using the default template to conduct tests on the effects of implementing AMS into our current missions.

**Comment by freghar** [comment]
_From @JamesTheClarke on October 4, 2015 11:22_

If you need help setting up the mission, especially the ladouts send me a list of items you require to spawn on which classes and I'll get it done.

**Comment by freghar** [comment]
_From @Abuk007 on October 13, 2015 0:24_

Are there plans to test out AMS with Revive/CPR system enabled? Or do you plan to do that after template implementation?

**Comment by freghar** [comment]
_From @CntoDev on October 13, 2015 0:31_

Due to the gameplay changes that will occur if we use the AMS as you describe being far greater than simply improving the medical role, we will be testing very simple core functionality and nothing more.

Our aim in these tests are to make the medic actually useful and interesting to play in operations, not to introduce realism or dependency to the role.

__SIGNED BY: @CodyBurton

**Comment by freghar** [comment]
_From @Abuk007 on October 18, 2015 1:33_

Revive/CPR is not only about realism, but an important mechanic to look at, considering people getting killed too quickly by AI it might be a good limiting factor, making buddy teams even more important and bleeding the main cause of death. If we are to switch to AMS in near future, we need to find a proper solution. So as you said - something that is interesting, useful and makes medic role more important - which revival system is supposed to improve on.

I believe it is worth testing.

There are 2 core questions regarding AMS when discussing its implementation:
Will we use AMS?
Will we use revival/CPR mechanic?

As those 2 changes have major impact on the playstyle.

The rest will be a matter of selecting proper and balanced medical loadout, as well as adjusting our playstyle to it.

**Comment by freghar** [comment]
_From @ghost on October 25, 2015 17:40_

Redacted my past comment on the revive system, but we will still be implementing it into the tests.

Also, uploading the mission file now.

**Comment by freghar** [comment]
_From @TRyujin on October 25, 2015 23:13_

Proposed changes:
**Revive Settings**
- Enable Revive - Players only


**Advanced Medical Settings**
- Allow PAK - Anyone
- Condition PAK - Stable
- Locations PAK - Anywhere



Regarding equipment changes:
**For troops**
- 10x type elastic bandage
- 1x Tourniquete


**For medics**
- 60x type elastic bandage
- 6x 1000ml Blood IV
- 6x the atropine amount



Is it possible to increase PAK treatment time? 2 minutes minimum.

**Comment by freghar** [comment]
_From @JamesTheClarke on October 26, 2015 11:58_

I don't mind using the revive option for a while, it was very enjoyable yesterday. But after a month or two we should take a look at if and how this change affects player decision making.

I have a lingering fear that once players fully understand the system, especially how to properly and quickly patching someone up to prevent them from bleeding out, that it can affect players making bad decisions. Meaning people being less careful in their approach, counting on not being able to die right away.

In an ideal scenario I'd wish for the reviving only to be triggered by calibres up to and including 7.62mm, and anything beyond that having the ability to insta-kill - especially explosions and .50 cals.

The advanced medical system deserves a "high priority" tag and should get priority treatment when looking for testing event dates. It will really bring our game-play to a new level.

Looking forward to the third testing round!

**Comment by freghar** [comment]
_From @ghost on October 26, 2015 17:13_

Scheduled next testing sessions for the 8th of November, will upload a new test mission with these settings.

**Comment by freghar** [comment]
_From @ghost on November 8, 2015 16:47_

Updated testing mission file, uploaded to server.

edit: test

**Comment by freghar** [comment]
_From @chrisleaptrott on November 22, 2015 0:10_

Hey guys,
What's the status on this? Asking due to Mission Making considerations for the future! Thanks.

**Comment by freghar** [comment]
_From @ghost on November 22, 2015 1:16_

Final scheduled test for this is the 20th or the 27th, details will be posted closer to the date.

**Comment by freghar** [comment]
_From @chrisleaptrott on November 22, 2015 10:2_

So, is it safe to assume that implementation will be sometime in January, if green-lit?

**Comment by freghar** [comment]
_From @TRyujin on November 22, 2015 10:18_

It all depends on the results of tests and overall conclusion. Hopefully we won't have to adjust settings further which would require further testing. It also depends on the final vote, and if successful on time necessary to prepare the documentation which would ensure fluid transition for members into this complex system.

**Comment by freghar** [comment]
_From @chrisleaptrott on November 22, 2015 10:26_

That's what I mean by 'green-lit'. Just trying to get an ETA if all things look good. If its not until after December then no worries.

**Comment by freghar** [comment]
_From @ghost on November 22, 2015 10:26_

If we cannot get this done within one last testing session, I will be prioritizing other time-sensitive tasks above the Ace3 AMS. We will not be scrapping it entirely, but other testing sessions will need to be taken much later down the line.

The final batch of testing, as mentioned before, will be near the end of December. If all is good in that test, we will be pushing to get this out **before the start of February**. (once the Dec-Jan campaign is finished)

I hope this helps, if you have any other questions don't hesitate to give me a shout.

Edit: This will now be scheduled for before March

**Comment by freghar** [comment]
_From @chrisleaptrott on November 22, 2015 10:29_

Perfect, that's all I needed to know.

**Comment by freghar** [comment]
_From @Abuk007 on December 7, 2015 13:25_

I believe we should be pushing this live after Hiatus, so we give a good reason for people to discover new gameplay with fresh mind after break.

**Comment by freghar** [comment]
_From @JamesTheClarke on December 7, 2015 13:35_

I think before we can push this again we need to wait till ACE catches up properly to fix the stamina system and rebalance the wounding / medical system. @chrisleaptrott informed me that since the update 1.54 people die like flies even more than ever before.

This is probably also a combination issue of RHS not having updated their vests / defense values to match the new vanilla damage handlers.

A quick temporary fix could be to slightly lower AI accuracy in r0b6 or r0b7 until the situation stabilises again.

**Comment by freghar** [comment]
_From @chrisleaptrott on December 7, 2015 13:49_

r0b6 is out, so forward the changes to 7. And yeah, its pretty much as if people didn't have vests at all, so I've been down-grading the AI's values a tad in MCC's control panel to compensate. Nothing major; if Friday's op was any indication, they're still deadly.

I do agree with Abuk, though, we should get this going soon after the break so we can adapt with relatively fresh minds to the new concepts and such.

I'll look into the promised hotfixed coming through the RHS and ACE 3 pipeline.

**Comment by freghar** [comment]
_From @JamesTheClarke on December 7, 2015 13:55_

Just a short answer to clarify: the edits on the AI skill need to be mainly done within F3 as it does assign the AI skill by default to all units spawned via MCC, Zeus or Ares **unless** the GM opens MCC settings and re-confirms the standard AI skill given by MCC. This would override the more detailed AI skill given by F3.

**Comment by freghar** [comment]
_From @chrisleaptrott on December 7, 2015 14:0_

Precisely.

If you can edit them to the desired temporary settings for now on the r0b7 'development' template and push that change that would be a much better 'fix' for us between now and when RHS and ACE fix things.

**Comment by freghar** [comment]
_From @chrisleaptrott on December 11, 2015 16:46_

With the RHS fix coming, I think it would be safe to take this up following the Hiatus. Regardless, I couldn't see it hurting us with things even at current. I'd say we do a final test during the Hiatus and try and get this in...part of my "Fresh year, fresh stuff" kind of mentality. I don't see any buzz from ACE about working on the stamina system further, so we'll probably end up going with the awesome mod Freghar built for us...but we'll see.

**Comment by freghar** [comment]
_From @TRyujin on December 11, 2015 17:5_

Before any further advanced medical system tests, can we please conclude what the module settings should be?

On last test, we were testing under assumption my suggestions from 26th of October were active. At the end it became apparent we were playing with some other settings.

**Comment by freghar** [comment]
_From @chrisleaptrott on December 11, 2015 17:17_

If Cody changed the settings, he did that of his own accord without any input from any of us. That's another addition to the "Cody knows best" file that we can come back and fix.

Ryujin, I think you are the one who has put the most effort and time into everything with this, so your suggestions are the ones with the most weight. ADV medical shouldn't have been pushed to the back burner as it was, especially given what other frivolous mods he had up for testing. I think that we can give this another go with your suggested settings and see if we like it or not.

Community feedback has been positive the whole while, and things showed great promise and then seemed to halt due to whatever machinations we can now correct.

@TRyujin, can you please list out your suggestions here and I'll get a test up for this next Sunday or sometime soon following.

**Comment by freghar** [comment]
_From @TRyujin on December 27, 2015 14:41_

Let's change this to low priority. We have more important testings and other community projects that supersede AMS.

**Comment by freghar** [comment]
_From @chrisleaptrott on December 27, 2015 15:24_

I agree. With RHS's HP fix we're not looking to rush to fix things.

**Comment by freghar** [comment]
_From @Abuk007 on January 24, 2016 2:20_

What about this? Im pretty sure this should be of a high priority task. We had some tests, lets not leave it on a backburner too long. This is on the same importance level as ASR or ALIVE/DAC.

**Comment by freghar** [comment]
_From @CNTODachi on March 28, 2016 12:23_

I know this subject isn't quite dead but I would like to get it going again so assigning myself. I will post the exact settings I will use for testing and possible dates for those tests.

Also seeing as my new training template has the new medical training area might I suggest that I create various different versions with advanced enabled with different settings and advanced enabled along with revive module and we as devlopment members test ourselves to see which feels right and then set up a mini op or even better a training with these settings enabled?
